packages/tables/backend/sirius-components-collaborative-tables/src/main/java/org/eclipse/sirius/components/collaborative/tables/api/ITableQueryService.java

```diff
@@ -33,4 +33,27 @@ public interface ITableQueryService {
 
     Optional<Column> findColumnById(Table table, UUID columnId);
 
+    /**
+     * Implementation which does nothing, used for mocks in unit tests.
+     *
+     * @author frouene
+     */
+    class NoOp implements ITableQueryService {
```

Comment:
Given how we test our code today, there shouldn't be any reason to create a new NoOp class anymore.